### PR TITLE
update ruby-style.yml from backend

### PR DESCRIPTION
**Pull request by owensims1**
Copies our source-of-truth ruby-style.yml from backend up here for use across repos

### CODE REVIEW

File: ruby/.ruby-style.yml [hunk 1]
```diff
@@ -635,115 +635,115 @@ Style/SignalException:
 Style/SingleLineBlockParams:
   Description: 'Enforces the names of some block params.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#reduce-blocks'
-  Enabled: true
+  Enabled: false
```
Comment (matteddy) [inline ruby/.ruby-style.yml]: Why is this disabled? 🤔 Did they just change the defaults?

Comment (gmkohler) [inline ruby/.ruby-style.yml]: which one is this?

File: ruby/.ruby-style.yml [hunk 2]
```diff
@@ -872,7 +872,12 @@ Style/WhileUntilModifier:
 Style/WordArray:
   Description: 'Use %w or %W for arrays of words.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#percent-w'
-  Enabled: true
+  Enabled: false
```
Comment (matteddy) [inline ruby/.ruby-style.yml]: 🤔

File: ruby/.ruby-style.yml [hunk 3]
```diff
@@ -908,14 +913,13 @@ Metrics/CyclomaticComplexity:
...
   Description: 'Limit lines to 100 characters.'
-  StyleGuide: 'Avoid lines longer than 100 characters'
   Max: 100
   Enabled: true
 
 Metrics/MethodLength:
   Description: 'Avoid methods longer than 10 lines of code.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#short-methods'
-  Enabled: true
+  Enabled: false
```
Comment (gmkohler) [inline ruby/.ruby-style.yml]: Is this just cause we break it a lot?

Comment (owensims1) [inline ruby/.ruby-style.yml]: Yes, I disabled it a while ago -- https://github.com/zestyzesty/backend/commit/477782d78b37a6168e0485b761884a80be7fc645

@matteddy brought up a good point that we should do an audit. Part of that should be centralizing the source, which I'm doing here.

**Pull request merged**